src/main/resources/hudson/plugins/gradle/injection/init-script.gradle

```diff
@@ -48,7 +48,7 @@ def BUILD_SCAN_PLUGIN_CLASS = 'com.gradle.scan.plugin.BuildScanPlugin'
 def GRADLE_ENTERPRISE_PLUGIN_ID = 'com.gradle.enterprise'
 def GRADLE_ENTERPRISE_PLUGIN_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterprisePlugin'
 def GRADLE_ENTERPRISE_EXTENSION_CLASS = 'com.gradle.enterprise.gradleplugin.GradleEnterpriseExtension'
-def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI-auto-injection'
+def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI auto injection'
```

Comment:
```suggestion
def CI_AUTO_INJECTION_CUSTOM_VALUE_NAME = 'CI auto-injection'
```

Reply:
See https://gradle.slack.com/archives/C03G38H88P4/p1666364657763789?thread_ts=1666352048.466839&cid=C03G38H88P4

Reply:
Revoking this as discussed.